forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -17,15 +17,25 @@ use forest_key_management::json::KeyInfoJson;
 use forest_rpc_client::wallet_ops::*;
 use forest_shim::address::{Address, Protocol};
 use forest_utils::io::read_file_to_string;
+#[allow(unused_imports)]
 use fvm_shared::{
     crypto::signature::{Signature, SignatureType},
     econ::TokenAmount,
 };
-use num::BigInt;
 use rpassword::read_password;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+#[allow(clippy::enum_variant_names)]
+enum FormattingMode {
+    ExactFixed,
+    NotExactFixed,
```

Comment:
I don't want to introduce new naming, as it would only confuse me. Also this won't fix the clippy warning, as is is caused by all enum variant be suffixed by `Fixed`.

Reply:
Why would it confuse you? I mean, `not exact` is kind of the same as `rounded`, no?

It would fix the clippy warning if you switch places. So `FixedExact, NotFixedRounded` etc.